### PR TITLE
add aimodelshare meta.yaml

### PR DESCRIPTION
**Pull request by raudipra**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by raudipra** [comment]
@conda-forge/help-python how to exclude win_64 win check? I already defined the selector to only support unix os.

File: recipes/aimodelshare/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 36b35968e0605fa57610eee39b568a491666ed4d76b5287ea8d276fb83294dd4
+
+build:
+  skip: True  # [unix]
```
Comment (chrisburr) [inline recipes/aimodelshare/meta.yaml:13]: The selectors make this line only appear if the expression evaluates to true so you want:
```suggestion
  skip: True  # [win]
```

**Comment by BastianZim** [comment]
Just to add: If the only reason that you're excluding windows is the missing dependency then you don't need to exclude windows and can keep this package `noarch`. The conda solver will inform the user during installation that this package cannot be installed due to missing dependencies and the failure here can be ignored.

**Comment by raudipra** [comment]
> Just to add: If the only reason that you're excluding windows is the missing dependency then you don't need to exclude windows and can keep this package `noarch`. The conda solver will inform the user during installation that this package cannot be installed due to missing dependencies and the failure here can be ignored.

The missing dependency will affect our features as well (e.g. TF2 -> TF1) so at this time we think it's better to exclude them for the time being.

**Comment by BastianZim** [comment]
I'm not sure if I'm following. It seems like you're excluding windows because the dependency is not available, is that correct? In that case you do not need to exclude it, the solver will take care of it.

**Comment by raudipra** [comment]
> I'm not sure if I'm following. It seems like you're excluding windows because the dependency is not available, is that correct? In that case you do not need to exclude it, the solver will take care of it.

Ah my bad, got it.

**Comment by BastianZim** [comment]
Basically, you just need to make sure that the dependencies are available on Linux, since that is where the package is being built. The solver will then only allow the installation on platforms where all dependencies are available and if they're not, everybody can submit them so that this package can be installed on these platforms. This makes it easier for the maintainer because you don't need to take care of all dependencies on all platforms.

**Comment by raudipra** [comment]
> Basically, you just need to make sure that the dependencies are available on Linux, since that is where the package is being built. The solver will then only allow the installation on platforms where all dependencies are available and if they're not, everybody can submit them so that this package can be installed on these platforms. This makes it easier for the maintainer because you don't need to take care of all dependencies on all platforms.

I've checked it on local osx and it is all available but haven't checked it on linux, isn't this linux_64 build supposed to check that for me?

thanks for the explanation that makes sense!

**Comment by BastianZim** [comment]
Yes if the build passes here you‘re good. By checking, I just meant that they need to be available when you’re submitting the package, you don’t need to manually check every single one.

**Comment by raudipra** [comment]
@conda-forge/help-python yo, it's ready

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
```
Comment (BastianZim) [inline LICENSE:1]: This can be excluded, the license is already included in the package.

File: recipes/aimodelshare/LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015-2022, conda-forge
```
Comment (BastianZim) [inline recipes/aimodelshare/LICENSE.txt:1]: Same here.

File: recipes/aimodelshare/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,64 @@
+{% set name = "aimodelshare" %}
+{% set version = "0.0.94" %}
+
+package:
+  name: "{{ name|lower }}"
```
Comment (BastianZim) [inline recipes/aimodelshare/meta.yaml:5]:
```suggestion
  name: {{ name|lower }}
```

File: recipes/aimodelshare/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  version: "{{ version }}"
```
Comment (BastianZim) [inline recipes/aimodelshare/meta.yaml:6]:
```suggestion
  version: {{ version }}
```

File: recipes/aimodelshare/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```
Comment (BastianZim) [inline recipes/aimodelshare/meta.yaml:9]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipes/aimodelshare/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+  sha256: 36b35968e0605fa57610eee39b568a491666ed4d76b5287ea8d276fb83294dd4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (BastianZim) [inline recipes/aimodelshare/meta.yaml:15]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

File: recipes/aimodelshare/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+  
+requirements:
+  host:
+    - pip
+    - python >=3.6
```
Comment (BastianZim) [inline recipes/aimodelshare/meta.yaml:20]:
```suggestion
    - python >=3.7
```

File: recipes/aimodelshare/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+    - botocore ==1.21.2
+    - docker-py ==5.0.0
+    - keras2onnx >=1.7.0
+    - onnx >=1.9.0
+    - onnxconverter-common >=1.7.0
+    - onnxmltools >=1.6.1
+    - onnxruntime >=1.7.0
+    - pyjwt ==2.2.0
+    - pympler ==0.9
+    - python >=3.6
```
Comment (BastianZim) [inline recipes/aimodelshare/meta.yaml:33]:
```suggestion
    - python >=3.7
```

File: recipes/aimodelshare/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+    - seaborn >=0.11.2
+    - shortuuid >=1.0.8
+    - skl2onnx >=1.8.0
+    - tensorflow >=2.5.0
+    - tf2onnx
+    - pytorch >=1.8.1
+    - urllib3 ==1.25.11
+    - wget ==1.20.3
+    - xgboost >=0.90
+    - dill
```
Comment (BastianZim) [inline recipes/aimodelshare/meta.yaml:44]: I cannot find this in the reqs. Is that needed?

File: recipes/aimodelshare/meta.yaml [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "Deploy locally saved machine learning models to a live rest API and web-dashboard.  Share it with the world via modelshare.org"
+  doc_url: https://github.com/AIModelShare/aimodelshare/blob/master/README.md
+  dev_url: https://github.com/AIModelShare/aimodelshare
+
+extra:
+  recipe-maintainers:
+    - aimodelshare
```
Comment (BastianZim) [inline recipes/aimodelshare/meta.yaml:64]: We need some confirmation from them that they're willing to be a maintainer.

**Review by BastianZim — CHANGES_REQUESTED** [review]

File: recipes/aimodelshare/meta.yaml (hunk 10 above)
Comment (raudipra) [reply recipes/aimodelshare/meta.yaml:44]: yes

File: recipes/aimodelshare/meta.yaml (hunk 11 above)
Comment (raudipra) [reply recipes/aimodelshare/meta.yaml]: @mikedparrott mike we need your confirmation. thanks!

Comment (mikedparrott) [reply recipes/aimodelshare/meta.yaml]: Yes.  Thanks, I confirm that I will be the maintainer of aimodelshare.

**Comment by raudipra** [comment]
@conda-forge/help-python it's ready.

**Comment by BastianZim** [comment]
Something is still wrong with the licenses. The file list should only list the meta.yaml.

File: LICENSE.txt [hunk 12]
```diff
@@ -24,4 +24,4 @@ DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
 SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
 CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
 OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```
Comment (BastianZim) [inline LICENSE.txt:27]: Please revert this change as well.

**Comment by raudipra** [comment]
@conda-forge/help-python it's finally ready.

**Comment by BastianZim** [comment]
Let me ping that team for you for a final check since you're not a part yet of conda-forge. @conda-forge/help-python

**Comment by raudipra** [comment]
@BastianZim how long does it take for the team to confirm / upload the package to the conda-forge channel?

File: recipes/aimodelshare/meta.yaml [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - astunparse ==1.6.3
```
Comment (hmaarrfk) [inline recipes/aimodelshare/meta.yaml:22]: I'm not against letting people shoot them selves in the foot, but equality pinnings should not be used for packages, rather they should be used in your environment files. can you consider releasing these as lower bound?

File: recipes/aimodelshare/meta.yaml [hunk 14]
```diff
@@ -0,0 +1,65 @@
...
+  license_family: MIT
+  license_file: LICENSE
+  summary: "Deploy locally saved machine learning models to a live rest API and web-dashboard.  Share it with the world via modelshare.org"
+  doc_url: https://github.com/AIModelShare/aimodelshare/blob/master/README.md
+  dev_url: https://github.com/AIModelShare/aimodelshare
+
+extra:
+  recipe-maintainers:
+    - raudipra
+    - mikedparrott
```
Comment (hmaarrfk) [inline recipes/aimodelshare/meta.yaml:65]: Can you kindly ask this person to confirm they want to maintain this?

Comment (BastianZim) [reply recipes/aimodelshare/meta.yaml:65]: https://github.com/conda-forge/staged-recipes/pull/17954#discussion_r800239478 Already checked :)

File: recipes/aimodelshare/meta.yaml (hunk 13 above)
Comment (raudipra) [reply recipes/aimodelshare/meta.yaml:22]: I am sorry but I don't think it is possible for now, several time encountered issues on latest version therefore we deliberately specify the working version.

Comment (BastianZim) [reply recipes/aimodelshare/meta.yaml:22]: Ok as long as you're aware of the limitations that imposes, I think we can merge this.

**Comment by raudipra** [comment]
@conda-forge-admin, please update team
